<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitReportReporter -->
<testsuite hostname="mac-PC" name="com.mac.example.Seltest" tests="1" failures="1" timestamp="20 Jan 2013 17:30:50 GMT" time="0.000" errors="0">
  <testcase name="testDataProviderExample" time="0.000" classname="com.mac.example.Seltest">
    <failure type="org.testng.TestNGException" message="
Method public void com.mac.example.Seltest.testDataProviderExample(java.lang.String,java.lang.String,java.lang.String,java.lang.String) throws java.lang.Exception requires a @DataProvider named : DP1">
      <![CDATA[org.testng.TestNGException: 
Method public void com.mac.example.Seltest.testDataProviderExample(java.lang.String,java.lang.String,java.lang.String,java.lang.String) throws java.lang.Exception requires a @DataProvider named : DP1
	at org.testng.internal.Parameters.findDataProvider(Parameters.java:265)
	at org.testng.internal.Parameters.handleParameters(Parameters.java:415)
	at org.testng.internal.Invoker.handleParameters(Invoker.java:1384)
	at org.testng.internal.Invoker.createParameters(Invoker.java:1075)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1180)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:128)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:111)
	at org.testng.TestRunner.privateRun(TestRunner.java:767)
	at org.testng.TestRunner.run(TestRunner.java:617)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:334)
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:329)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:291)
	at org.testng.SuiteRunner.run(SuiteRunner.java:240)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1203)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1128)
	at org.testng.TestNG.run(TestNG.java:1036)
	at org.testng.remote.RemoteTestNG.run(RemoteTestNG.java:111)
	at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:204)
	at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:175)
]]>
    </failure>
  </testcase> <!-- testDataProviderExample -->
</testsuite> <!-- com.mac.example.Seltest -->
